api/app/responses/tests.py

```diff
@@ -76,7 +76,7 @@ def _seed_data(self):
             datetime.now(), datetime.now(), datetime.now(), datetime.now())
         _add_object_to_db(self.test_event)
         self.test_form = ApplicationForm(
-            self.test_event.id, True)
+            self.test_event.id, True, False)
```

Comment:
Please change these to all your new method instead: `self.create_application_form(...)` - that way, next time someone makes a change to ApplicationForm, they don't have to update all these tests again...
(same comment applies to the other tests)